### ISSUE TITLE
Custom feature points/matches

### ISSUE THREAD

**Issue by SamuelJamesFisher** [AIBluefisher/GraphSfM#63]
Would it be possible to use a Colmap database that already contains features and matches?

**Comment by AIBluefisher** [comment]
Yes. However, since COLMAP's database does not store relative poses, you need to modify the code that leverages relative poses to filter outliers. Let me know if you need further instructions.
